client/state/data-layer/wpcom/plans-request.js

```diff
@@ -0,0 +1,27 @@
+import wpcom from 'lib/wp';
```

Comment:
I do think naming and folder structure of the files here is an important design decision, since we'll need to account for its scaling while ensuring they remain discoverable. I could see it being a `kebab-case` representation of the action type's handler, though in that case exporting a default of `key: value` is maybe not the best way, since we'd only one a max of one key per file. 

Instead, could we take inspiration from Redux's `combineReducers` and map the action type handler at the point of combining? Something like:

__state/data-layer/wpcom/plans-request.js__

```js
export default [
	( { dispatch } ) =>
		wpcom /* ... */
];
```

__state/data-layer/wpcom/index.js__:

```js
import plansRequest from './plans-request';

export default combineHandlers( {
	[ PLANS_REQUEST ]: plansRequest
} );
```